microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -29,44 +36,24 @@ class Dpdk(TestSuite):
              with one interface for management.
             More detailes refer https://docs.microsoft.com/en-us/azure/virtual-network/setup-dpdk#prerequisites # noqa: E501
         """,
+        priority=2,
         requirement=simple_requirement(
             min_nic_count=2,
             network_interface=Sriov,
         ),
-        priority=1,
     )
     def check_dpdk_build(self, node: Node, log: Logger) -> None:
-        network_interface_feature = node.features[NetworkInterface]
-        sriov_is_enabled = network_interface_feature.is_enabled_sriov()
-        log.info(f"Verify SRIOV is enabled: {sriov_is_enabled}")
-        assert_that(sriov_is_enabled).described_as(
-            "SRIOV was not enabled for this test node."
-        ).is_true()
-
-        # dump some info about the pci devices before we start
-        lspci = node.tools[Lspci]
-        log.info(f"LSPCI Info:\n{lspci.run().stdout}\n")
-
-        # enable hugepages (required by dpdk)
-        self._hugepages_init(node)
-        self._hugepages_enable(node, log)
-
-        # initialize testpmd tool (installs dpdk)
-        testpmd = DpdkTestpmd(node)
-        testpmd.install()
-
-        # initialize node nic info class (gathers info about nic devices)
-        node_nic_info = NodeNicInfo(node)
-        assert_that(len(node_nic_info)).described_as(
-            "Test needs at least 2 NICs on the test node."
-        ).is_greater_than_or_equal_to(2)
-        log.info(node_nic_info)
+        # setup and unwrap the resources for this test
+        test_kit = DpdkTestKit(node, log)
+        test_kit.initialize()
+        node_nic_info, testpmd = test_kit.node_nic_info, test_kit.testpmd
 
         # grab a nic and run testpmd
         test_nic = node_nic_info.get_nic(node_nic_info.get_upper_nics()[-1])
         vdev_type = "net_vdev_netvsc0"
         testpmd_include_str = test_nic.testpmd_include(vdev_type)
-        testpmd_output = testpmd.run_with_timeout(testpmd_include_str, 30)
+        testpmd_cmd = testpmd._generate_testpmd_command(testpmd_include_str, "txonly")
+        testpmd_output = testpmd.run_for_n_seconds(testpmd_cmd, 30)
```

Comment:
30 seconds is pretty long. Can it be tested in shorter time, how about 5 seconds?

Reply:
We need to gather the packets-per-second so I'd rather have a little more time than not enough. I'll change to 10.